pootle/apps/pootle_store/utils.py

```diff
@@ -104,9 +104,7 @@ def add(self, unit, translation, user=None, touch=True,
                 mt_similarity=mt_similarity,
             )
             sub.save()
-
-            if touch:
-                unit.save()
+            unit.save()
```

Comment:
actually it seems not i think - i think its there for when a suggestion is added during eg unit.update

it would be good to rethink this pattern a bit - but for now lets keep